cmd/temporalite/main.go

```diff
@@ -42,6 +42,7 @@ const (
 	headlessFlag           = "headless"
 	ipFlag                 = "ip"
 	uiIPFlag               = "ui-ip"
+	codecEndpointFlag      = "codec-endpoint"
```

Comment:
Nit: Can we call this `ui-codec-endpoint`?

Reply:
Indeed, much more consistent, updated!